packages/client/src/objectSet/createObjectSet.ts

```diff
@@ -230,14 +230,23 @@ export function createObjectSet<Q extends ObjectOrInterfaceDefinition>(
       properties,
       listener,
     ) => {
+      const referenceProperties = properties.filter(p => {
+        objectType.__DefinitionMetadata?.properties[p].type
+          === "geotimeSeriesReference";
```

Comment:
What's the best way to enforce this type? Should we expose an from WirePropertyTypes in API, or should this be maintained locally.